package.json

```diff
@@ -47,12 +47,16 @@
     "node-libs-browser": "^0.5.2",
     "phantomjs": "^1.9.17",
     "raw-loader": "^0.5.1",
-    "react": "^0.13.3",
+    "react": "^0.14.0-rc1",
+    "react-dom": "^0.14.0-rc1",
     "webpack": "^1.10.1",
     "webpack-dev-server": "^1.6.6"
   },
+  "dependencies": {
+    "highlight.js": "^8.4.x"
+  },
   "peerDependencies": {
-    "highlight.js": "^8.4.x",
-    "react": "*"
+    "react": "^0.14.0-rc1",
```

Comment:
Yes, I will review and merge this PR

On Wed, Nov 4, 2015 at 7:44 PM, Marcel Panse <notifications@github.com>
wrote:

> In package.json
> <https://github.com/akiran/react-highlight/pull/9#discussion_r43884411>:
>
> >    "peerDependencies": {
> > -    "highlight.js": "^8.4.x",
> > -    "react": "*"
> > +    "react": "^0.14.0-rc1",
>
> Done
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/akiran/react-highlight/pull/9/files#r43884411>.
>